ern-container-gen/src/addContainerMetadata.ts

```diff
@@ -6,8 +6,10 @@ import fs from 'fs-extra'
 export async function addContainerMetadata(conf: ContainerGeneratorConfig) {
   const metadata: ContainerMetadata = {
     ernVersion: Platform.currentVersion,
-    jsApiImpls: conf.composite.getJsApiImpls().map(j => j.toString()),
-    miniApps: conf.composite.getMiniApps().map(m => m.packagePath.toString()),
+    jsApiImpls: (await conf.composite.getJsApiImpls()).map(j => j.toString()),
+    miniApps: (await conf.composite.getMiniApps()).map(m =>
```

Comment:
Just wondering: Is this general cleanup or related to the other changes?

Reply:
Related to other changes. The two functions that are called where synchronous prior to this PR, and were changed to asynchronous ones, as part of a change in this PR, thus the need to add the `await` keyword when calling these functions.